app/lib/check_answers_fields.yml

```diff
@@ -108,55 +108,57 @@ sections:
   - label: assets
     subsections:
       - label: property
+        screen: property
         fields:
           - type: select
             attribute: property_owned
-            screen: property
+            disputed_attribute: house_in_dispute
           - type: money
             attribute: house_value
-            screen: property_entry
```

Comment:
An unfortunate side effect of these `screen` removals is the presenter no longer understands that if the `property_entry` screen is to be skipped (i.e. the user has said they do not own their house), these fields should always be skipped. Leading to:
<img width="488" alt="Screenshot 2022-11-09 at 16 06 45" src="https://user-images.githubusercontent.com/113888736/200881061-257f5aef-bb5f-4361-839e-202ffe11d78e.png">

Reply:
(this is my own damn fault for making these fields 'clever'. The solution might be to leave the link in between fields and screens in the yml file, for the purposes of knowing which fields to omit based on which screens to omit. But then _not_ use the field's `screen` attribute to construct a link.